### PR TITLE
Bug 1132230 - Do not mirror icons by default, except forward on settings

### PR DESCRIPTION
**Pull request by autra**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[autra Augustin Trancart (autra)](https://github.com/autra) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8f6cd25dad51)

File: apps/settings/style/icons.css [hunk 1]
```diff
@@ -295,6 +295,7 @@ aside.connecting {
 html[dir="rtl"] a.menu-item:after {
   left: 0.3rem;
```
Comment (wilsonpage) [inline apps/settings/style/icons.css]: `left: -0.9rem;`

Comment (wilsonpage) [inline apps/settings/style/icons.css]: The icon should align with the above/below dividing line.

**Pull request closed without merging**